android-json-form-wizard/src/main/res/layout/native_json_form_fragment.xml

```diff
@@ -1,17 +1,63 @@
+        android:id="@+id/scroll_view"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        android:orientation="vertical">
+
+        <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+            android:id="@+id/main_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:paddingStart="@dimen/default_left_margin"
+            android:paddingEnd="@dimen/default_right_margin"
+            android:paddingTop="8dp">
 
-    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        android:id="@+id/main_layout"
+        </LinearLayout>
+    </ScrollView>
+    <LinearLayout
+        android:id="@+id/bottom_navigation_layout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="vertical"
-        android:paddingStart="@dimen/default_left_margin"
-        android:paddingEnd="@dimen/default_right_margin"
-        android:paddingTop="8dp">
+        android:layout_height="wrap_content"
+        android:visibility="gone">
+        <Button
+            android:id="@+id/previous_button"
+            style="@style/Widget.AppCompat.Button.Borderless"
+            android:layout_width="0dp"
+            android:layout_weight="1"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/bottom_navigation_margin"
+            android:layout_marginTop="@dimen/bottom_navigation_margin"
+            android:layout_marginLeft="@dimen/bottom_navigation_margin"
+            android:paddingEnd="@dimen/bottom_navigation_padding"
+            android:paddingStart="@dimen/bottom_navigation_padding"
+            android:text="@string/previous"
+            android:textAllCaps="true"
+            android:textAppearance="@style/TextAppearance.AppCompat.Button"
+            android:textColor="#737373"
+            android:textSize="20sp" />
 
+        <Button
+            android:id="@+id/next_button"
+            style="@style/Widget.AppCompat.Button.Borderless"
+            android:layout_width="0dp"
+            android:layout_weight="1"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/bottom_navigation_margin"
+            android:layout_marginTop="@dimen/bottom_navigation_margin"
+            android:layout_marginRight="@dimen/bottom_navigation_margin"
+            android:background="@drawable/btn_bg"
+            android:paddingEnd="@dimen/bottom_navigation_padding"
+            android:paddingStart="@dimen/bottom_navigation_padding"
+            android:text="@string/next"
+            android:textAllCaps="true"
+            android:textAppearance="@style/TextAppearance.AppCompat.Button"
+            android:textColor="#FFFFFF"
+            android:textSize="20sp" />
```

Comment:
Add a visibly of gone so that button is enabled only step has a tag of next

Reply:
Done.